pageserver/src/http/routes.rs

```diff
@@ -395,6 +395,9 @@ async fn timeline_create_handler(
                     format!("{err:#}")
                 ))
             }
+            Err(e @ tenant::CreateTimelineError::AncestorNotActive) => {
+                json_response(StatusCode::INTERNAL_SERVER_ERROR, HttpErrorBody::from_msg(e.to_string()))
```

Comment:
Well... It's only retryable if the upload was still in progress. If the timeline would become stopping, you'd get the same error, but perhaps the error would eventually turn into "Cannot branch off timeline that's not present in the pageserver." This would be 500.

Reply:
[6b08e9d](https://github.com/neondatabase/neon/pull/5484/commits/6b08e9d219f16712e53a2a41def779bdc2fb47e7)